src/assets/server/rest/v1/schemas/chargingstation/chargingstation-qrcode-download.json

```diff
@@ -0,0 +1,32 @@
+{
+  "title": "Download QRCode of Charging Stations",
+  "type": "object",
+  "properties": {
+    "ChargingStationID": {
+      "$ref": "chargingstation.json#/definitions/id"
+    },
+    "ConnectorID": {
+      "$ref": "chargingstation.json#/definitions/connectorIDs"
+    },
+    "SiteAreaID": {
+      "$ref": "common.json#/definitions/ids"
+    },
+    "SiteID": {
+      "$ref": "common.json#/definitions/ids"
+    },
+    "ProjectFields": {
+      "$ref": "common.json#/definitions/projectFields"
+    }
```

Comment:
I think there is no projected fields in downloading the QR-Code as you get a PDF

Reply:
Indeed, removed